package.json

```diff
@@ -82,29 +82,18 @@
     "decache": "^4.1.0",
     "del": "^2.2.1",
     "gulp": "github:gulpjs/gulp#v4.0.0",
-    "gulp-clean-css": "^2.0.13",
     "gulp-concat": "^2.6.0",
     "gulp-filter": "^5.0.0",
     "gulp-install": "^0.6.0",
     "gulp-istanbul-report": "0.0.1",
     "gulp-json-editor": "^2.2.1",
-    "gulp-remote-src": "0.4.1",
     "gulp-rename": "^1.2.2",
     "gulp-shell": "^0.5.2",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-tslint": "^8.1.4",
     "gulp-typescript": "^3.1.4",
     "gulp-uglify": "^2.0.0",
     "istanbul": "^0.4.5",
-    "jasmine-core": "~2.4.1",
-    "karma": "^1.3.0",
```

Comment:
this was used for the frontend tests, if you plan on bringing that back, you will need to keep this.